jaeger-core/src/test/java/io/jaegertracing/internal/JaegerSpanTest.java

```diff
@@ -208,18 +208,6 @@ public void testWithoutTimestampsInaccurateClock() {
     assertEquals(10, jaegerSpan.getDuration());
   }
 
-  @Test
-  public void testSpanToString() {
-    JaegerSpan jaegerSpan = tracer.buildSpan("test-operation").start();
-    JaegerSpanContext expectedContext =  jaegerSpan.context();
-    JaegerSpanContext actualContext = TextMapCodec.contextFromString(expectedContext.contextAsString());
-
-    assertEquals(expectedContext.getTraceId(), actualContext.getTraceId());
-    assertEquals(expectedContext.getSpanId(), actualContext.getSpanId());
-    assertEquals(expectedContext.getParentId(), actualContext.getParentId());
-    assertEquals(expectedContext.getFlags(), actualContext.getFlags());
```

Comment:
removing it is not equivalent. But you can change L215 to use TextMapCodec's extract method.

Reply:
I am not sure if this still applies. See testSpanToString in `TextMapCodec`.

Reply:
Here's what I am thinking. The method is called `testSpanToString`. Such method should belong in SpanTest class, not the codecs. But it's not actually testing `span.toString()`, only `spanContext.contextAsString()`. So let's separate these.
  * to test `span.toString()`, we can simply compare with an exact literal string (including the operation name).
  * what is the contract of `contextAsString()`? Do we, strictly speaking, care if the output matches the output of the codec? What if we use a different codec in the future?  I think it mattered more when span context had the opposite `fromString()` method, but now `contextAsString()` seems merely a dependency of `span.toString()`, nothing more. Or is it used elsewhere?  If not, we can not test it at all (since it's covered by `span.toString()` test), and we can consider making it package viz.



Counter-proposal: keep the test here (since it's testing Span's behavior, not the codec), but

Reply:
But what? :)

Reply:
sorry, a left over